README.md

```diff
@@ -1494,6 +1494,15 @@ const Page = () => (
       jobLocationType="TELECOMMUTE"
       validThrough="2020-01-06"
       applicantLocationRequirements="FR"
+      experienceRequirements={{
```

Comment:
The shape of this object does differ from the schema outlined on [googles doc's](https://developers.google.com/search/docs/appearance/structured-data/job-posting#education-and-experience-properties-beta) just to keep all the experience information together.

Let me know if it would be better to use the same structure as the one used in the docs.